src/main/java/bio/terra/service/profile/flight/create/CreateProfileManagedResourceGroup.java

```diff
@@ -0,0 +1,38 @@
+package bio.terra.service.profile.flight.create;
+
+import bio.terra.common.iam.AuthenticatedUserRequest;
+import bio.terra.model.BillingProfileRequestModel;
+import bio.terra.service.profile.ProfileService;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.Step;
+import bio.terra.stairway.StepResult;
+import bio.terra.stairway.exception.RetryException;
+
+public class CreateProfileManagedResourceGroup implements Step {
+
+  private final ProfileService profileService;
+  private final BillingProfileRequestModel request;
+  private final AuthenticatedUserRequest user;
+
+  public CreateProfileManagedResourceGroup(
+      ProfileService profileService,
+      BillingProfileRequestModel request,
+      AuthenticatedUserRequest user) {
+    this.profileService = profileService;
+    this.request = request;
+    this.user = user;
+  }
+
+  @Override
+  public StepResult doStep(FlightContext flightContext)
+      throws InterruptedException, RetryException {
+    profileService.registerManagedResourceGroup(request, user);
```

Comment:
I had left this question in the ticket:
> What is the Sam endpoint’s behavior when a billing profile already exists for the MRG coordinates?

Looking at the documented responses, it will return a 409: https://sam.dsde-prod.broadinstitute.org/#/Azure/createManagedResourceGroup

Flagging this as I'm not sure if this will end up proving problematic in the current setup for our `AzureIntegrationTest`s.  Let's revisit that after the next round of changes.